data/NFTE/data.json

```diff
@@ -1,16 +1,16 @@
 {
-    "name": "NFTEarth",
+    "name": "NFTEarthOFT",
     "symbol": "NFTE",
     "decimals": 18,
-    "description": "NFTEarth is a marketplace for NFTs.",
+    "description": "NFTEarth is a decentralized NFT marketplace on built on Optimism and other networks designed for scalability.",
     "website": "https://nftearth.exchange",
     "twitter": "@NFTEarth_L2",
     "tokens": {
-      "ethereum": {
-        "address": "0x0f9b80fc3c8b9123D0aEf43Df58ebDBC034A8901"
-      },
```

Comment:
In the description then can you please clarify that this token is "bridged by LayerZero", and then also add
`"nobridge": true` to your data.json?  As an example, see Apt (which uses Wormhole): https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/master/data/APT/data.json

Reply:
Yes absolutely - since your question though, We've just been discussing as a team right now if we should deploy to Mainnet as well to have this available for users - is there a preference here towards a particular direction? 

We are building our own in-app bridge as well and so now thinking it may make more practical sense to have Mainnet available even tho we are aiming to get people off of it. Let me know, thank you.

Reply:
Hi @roberto-bayardo - the team decided to deploy the NFTE OFT token to Ethereum Mainnet in the end. Apologies for any confusion here, also appreciate your question on the matter as long-term we do think this is ultimately the best solution for onboarding users to L2 by also having the token available on Mainnet.

Reply:
The data.json file has been updated to reflect the accurate Ethereum Mainnet address `0x8c223a82e07fecb49d602150d7c2b3a4c9630310` - please let me know if you require anything else. Thank you.